.travis.yml

```diff
@@ -3,6 +3,8 @@ python: 2.7
 env:
     - TOX_ENV=py27
     - TOX_ENV=py33
+    - TOX_ENV=py34
+    - TOX_ENV=py35
```

Comment:
you may want to see the result of ``python --version`` at the end of each test job: https://travis-ci.org/novafloss/workalendar/builds/118435186